doc/contributing/documentation_guide.md

```diff
@@ -199,6 +199,36 @@ will be autolinked:
 If not, or if you suppress autolinking, consider forcing
 [monofont](rdoc-ref:RDoc::MarkupReference@Monofont).
 
+### Explicit Links
+
+When writing an explicit link, follow these guidelines.
+
+#### +rdoc-ref+ Scheme
+
+Use the +rdoc-ref+ scheme for:
+
+- A link in core documentation to other core documentation.
+- A link in core documentation to documentation in a standard library package.
+- A link in a standard library package to other documentation in that same
+  standard library package.
+
+See section "+rdoc-ref+ Scheme" in {Links}[https://docs.ruby-lang.org/en/master/RDoc/MarkupReference.html#class-RDoc::MarkupReference-label-Links].
```

Comment:
Isn't this a link in the core documentation to documentation in a standard library package, and therefore, shouldn't it use `rdoc-ref` according to this?

Reply:
I have not been able to get that to work.  Please make correction and merge.

Reply:
I'm not sure how to do so either.  Considering that, maybe we shouldn't be recommending use of rdoc-ref in a case where we don't know how to get it to work?

Reply:
@drbrain, can you help us out here?